package.json

```diff
@@ -67,7 +70,7 @@
     "prepare": "npm run compile",
     "pretest-only": "npm run compile",
     "posttest": "npm run lint",
-    "system-test": "echo no system tests 😱",
+    "system-test": "nyc mocha --no-timeouts build/system-test/system.js",
```

Comment:
`--no-timeouts` is bad bad bad.  If any test crashes, it will just exit the test and mark it as successful:
https://github.com/mochajs/mocha/issues/2537

I would pick a very high number you're comfortable with here, but don't use `--no-timeouts`.